src/Microsoft.AspNet.OData.Shared/DeltaOfTStructuralType.cs

```diff
@@ -84,12 +101,30 @@ public Delta(Type structuralType, IEnumerable<string> updatableProperties)
         /// names, including those of dynamic properties, are ignored.</param>
         /// <param name="dynamicDictionaryPropertyInfo">The property info that is used as dictionary of dynamic
         /// properties. <c>null</c> means this entity type is not open.</param>
+        public Delta(Type structuralType, IEnumerable<string> updatableProperties, PropertyInfo dynamicDictionaryPropertyInfo)
+            : this(structuralType, updatableProperties: updatableProperties, dynamicDictionaryPropertyInfo, edmType: null)
+        {
+
+        }
+
+        /// <summary>
+        /// Initializes a new instance of <see cref="Delta{TStructuralType}"/>.
+        /// </summary>
+        /// <param name="structuralType">The derived entity type or complex type for which the changes would be tracked.
+        /// <paramref name="structuralType"/> should be assignable to instances of <typeparamref name="TStructuralType"/>.
+        /// </param>
+        /// <param name="updatableProperties">The set of properties that can be updated or reset. Unknown property
+        /// names, including those of dynamic properties, are ignored.</param>
+        /// <param name="dynamicDictionaryPropertyInfo">The property info that is used as dictionary of dynamic
+        /// properties. <c>null</c> means this entity type is not open.</param>
+        /// <param name="edmType">Edm type of the TStructuralType</param>
         public Delta(Type structuralType, IEnumerable<string> updatableProperties,
-            PropertyInfo dynamicDictionaryPropertyInfo)
+            PropertyInfo dynamicDictionaryPropertyInfo, IEdmTypeReference edmType)
```

Comment:
>IEdmTypeReference edmType) [](start = 56, length = 26)

this is a public constructor on a public class -- adding a required constructor argument is a breaking change.

Reply:
this was an issue with how the change is shown in codeflow.  This is actually a new overload -- the previous overload still exists.

---
In reply to: [556078293](https://github.com/OData/WebApi/pull/2275#discussion_r556078293) [](ancestors = 556078293)